### PR TITLE
Update node-statsum for redeployability 

### PR DESCRIPTION
**Pull request by imbstack**
This allows it to take `rootUrl` instead of `baseUrl` and works in conjunction with taskcluster/taskcluster-auth#156.

It will need to wait until that is deployed to land. We should update our other statsum libs as well.

https://bugzilla.mozilla.org/show_bug.cgi?id=1460770

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,38 +5,26 @@
...
-    "babel-runtime": "^6.0.0",
     "debug": "^2.2.0",
     "get-stream": "^2.0.0",
     "got": "^5.0.0",
     "jsonwebtoken": "^5.7.0",
     "lodash": "^4.5.1",
-    "url-join": "^0.0.1",
-    "uuid": "^2.0.2",
-    "promise": "^7.1.1"
+    "taskcluster-lib-urls": "^1.0.0",
```
Comment (jonasfj) [inline package.json]: This shouldn't use tc-lib-urls....

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ let Statsum = require('statsum');
 // Create a client object
 let configurer = async (project) => { return {
   project: 'example-project',
-  baseUrl: 'https://example.com',
+  rootUrl: 'https://example.com',
```
Comment (jonasfj) [inline README.md]: This is not a TC thing... :)

Any place we use `node-statsum` we learn the `baseUrl` of the statsum server from `auth.statsumToken` which contains the `baseUrl`:
https://docs.taskcluster.net/reference/platform/taskcluster-auth/references/api#get-token-for-statsum-project

**Review by jonasfj — DISMISSED** [review]
Love the removal of babel, but this isn't as TC baseUrl thing.... so no need to clean up that.

File: README.md (hunk 2 above)
Comment (imbstack) [reply README.md]: It doesn't always come from `auth.statsumToken`! Not in auth itself anyway.

**Comment by djmitche** [comment]
All that's left here is de-bablification and comments indicating that baseUrl is not a baseUrl.

**Review by djmitche — APPROVED** [review]
weird, it seems dismissing jonas's review actually dismissed mine

**Comment by djmitche** [comment]
0.6.0

**Pull request merged**